src/main/java/JSON/JSONHelper.java

```diff
@@ -0,0 +1,29 @@
+package JSON;
+
+import lombok.SneakyThrows;
+import org.apache.http.HttpEntity;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class JSONHelper {
+
+    @SneakyThrows
+    public JSONObject convertResponseBodyIntoJSON(HttpEntity httpEntity) {
```

Comment:
I really like Jackson library to serialize or deserialize objects. I suggest to use it. Article that can be useful https://www.baeldung.com/jackson-object-mapper-tutorial

Reply:
I removed JsonHelper class. Instead of this I implemented through objectMapper